lib/textrazor/api_response.rb

```diff
@@ -0,0 +1,54 @@
+require 'json'
+
+module TextRazor
+
+  class ApiResponse
```

Comment:
Similar to `ApiRequest`, `Response` class was meant to take care of this responsibility. Any particular reason to do this.

Reply:
The `Response` class is heavily oriented towards parsing responses to analysis requests.

The more general `ApiResponse` can handle any kind of response from TextRazor. (Side note: `Response` inherits from `ApiResponse`)